vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -810,6 +837,60 @@ def get_moe_configs(
     return None
 
 
+def get_moe_wna16_block_config(config: Dict[str,
```

Comment:
nit: would be nice to leave comments for some of the cases in the config heuristics